yamlcrd/pkg/parser/testdata/simple1/input.yaml

```diff
@@ -0,0 +1,22 @@
+XNamespace:
```

Comment:
is this example using all the features you've implemented? I'm curious how some of the complicated corner-cases are going to turn out?

Reply:
So there's definitely some major cheating going on right now, things like `map(key=string,value=string)` and `int(min=0)` are hard-coded so we can iterate on the syntax before committing to it.

Are there any particular corner scenarios you have in mind?